examples/CustomEntitlementComputationSample/gradle/libs.versions.toml

```diff
@@ -2,7 +2,7 @@
 agp = "8.0.2"
 androidxNavigation = "2.5.3"
 kotlin = "1.7.20"
-purchases = "6.5.0"
+purchases = "6.8.0"
```

Comment:
not sure why this was 6.5.0

Reply:
I'm guessing since it is using the local version (in settings.gradle), it doesn't really matter?

Reply:
It is outdated on MagicWeatherCompose, which is what I used as a base project for the sample. I am not sure why it's outdated there, but I guess it's an issue in the releasing scripts

Reply:
Both magic weather sample apps don't update their version automatically... We definitely should fix that, even if it's just pointing to the latest snapshot...  I created a task [here](https://linear.app/revenuecat/issue/SDK-3242/auto-update-versions-in-magicweather-sample-apps-android).